plugin/vimconda.py

```diff
@@ -208,8 +208,12 @@ def get_conda_info_dict():
       "user_rc_path": "/Users/calebhattingh/.condarc"
     }
     """
-    output = vim_conda_runshell('$CONDA_EXE info --json')
-    return json.loads(output)
+    try:
+        output = vim_conda_runshell('$CONDA_EXE info --json')
+        return json.loads(output)
+    except CalledProcessError:
+        cmd = vim_conda_runshell('echo $CONDA_EXE').strip()
+        raise RuntimeError(f"$CONDA_EXE is not set to a valid conda executable ($CONDA_EXE='{cmd}')") from None
```

Comment:
f-strings are available only since Python 3.6, and I'm not sure if there are any users still running 3.5 (also, it is definitely not available on Python 2.7). It might be safer to just use old-school format. What do you think?

```suggestion
        raise RuntimeError(f"$CONDA_EXE is not set to a valid conda executable ($CONDA_EXE='{cmd}')".format(cmd)) from None
```

Reply:
Good point - I've got so used to them at this point I forgot they're not universal!